### PR TITLE
Collect tasks profile per site as well

### PR DESCRIPTION
**Pull request by komamitsu**
We've added `digdag profile` cli command https://github.com/treasure-data/digdag/pull/1533. But it collects and outputs only the whole summary of all the sites' tasks not grouping per site. When using Digdag with multiple sites, profiled metrics per site would be very useful information to see the performance and stability. This PR introduces the feature.

### CODE REVIEW

File: digdag-cli/src/main/java/io/digdag/cli/profile/TasksStats.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
+package io.digdag.cli.profile;
```
Comment (komamitsu) [inline digdag-cli/src/main/java/io/digdag/cli/profile/TasksStats.java:1]: Moved from TaskSummary

File: digdag-cli/src/main/java/io/digdag/cli/profile/TasksSummary.java [hunk 2]
```diff
@@ -101,35 +66,23 @@ public TasksSummary(
         this.execDurationMillis = execDurationMillis;
     }
 
-    static class Builder
+    interface Builder
```
Comment (komamitsu) [inline digdag-cli/src/main/java/io/digdag/cli/profile/TasksSummary.java:69]: This is abstracted for PropagatableTasksSummaryBuilder

**Review by yoyama — APPROVED** [review]
LGTM.

**Comment by komamitsu** [comment]
Thanks!

**Pull request merged**